pkg/controller/configmap/configmap_controller.go

```diff
@@ -137,6 +138,12 @@ func add(mgr manager.Manager, sharedInfo *sharedinfo.SharedInfo, r reconcile.Rec
 		return err
 	}
 
+	// Watch for changes to primary resource ClusterRole
+	err = c.Watch(&source.Kind{Type: &rbacv1.ClusterRole{}}, &handler.EnqueueRequestForObject{})
```

Comment:
Adding `byNameSpaceFilter(watchedNamespace)` as other watchers?

Reply:
clusterrole is a cluster level object, we should not add the namespace filter for it, otherwise, the configmap reconciler cannot receive the clusterrole event.